packages/wdio-allure-reporter/package.json

```diff
@@ -9,7 +9,7 @@
   "engines": {
     "node": ">=10.0.0"
   },
-  "types": "./allure-reporter.d.ts",
+  "types": "./build/index.d.ts",
```

Comment:
Can you verify that the generated types are backwards compatible?

Reply:
Is the test in `test:typings` enough? Should I verify with a standalone project as well?

Reply:
@christian-bromann This causes the conflicting node types error, because these 2 lines are autogenerated by tsc
```
/// <reference types="webdriver" />
/// <reference types="node" />
import WDIOReporter from '@wdio/reporter';
import Allure from 'allure-js-commons';
```

I believe on master, we've already have conflicting `@types/node`, i.e. multiple `@types/node` packages hoisted after lerna bootstrap. It is a timebomb that will explode when we get rid of providing `.d.ts` typings manually and ship the declaration generated by Typescript. 

In order to avoid this error, I will need to revert back to manual `.d.ts` file as main file for now.

Reply:
Another solution is we could upgrade to Allure 2 and use its typings instead.

Reply:
let's revert this change to unblock us and pick it up at a later stage.